data/changelog.txt

```diff
@@ -1055,6 +1077,11 @@ Running over large/hard objects in vehicles will damage your wheels
 Separate toolsets for things embedded in wood and metal structures
 Multiple simultaneous and devastating deep nerfs to fire vs monsters
 Make picking individual threads out of cotton scraps SUCK (for morale)
+Remove tool flags from razor blade I just added
+microlab alpha primer vault now has catalyst
+Change Throwing Assist CBM's enchant to use THROW_STR and THROW_DAMAGE instead of adding to skill
+Greatly reduce lone fruit trees in the forest
+Rebalance Aikido to emphasize surviving and escaping opponents
```

Comment:
```suggestion
Change Throwing Assist CBM's enchant to use THROW_STR and THROW_DAMAGE instead of adding to skill
Greatly reduce lone fruit trees in the forest
Rebalance Aikido to emphasize surviving and escaping opponents
```